### DISCUSSION TITLE
r114 linux virtual machine arcvm hammering CPU

### DISCUSSION THREAD

**Discussion by Coasie** [sebanc/brunch#1891] [General]
updated from r113 & CHROS 113 to r114 & CHROS 114, now linux virtual machine arcvm is using a lot of CPU, watching the task manager, it is often seen using 200 - >300% of CPU, and it makes the system very sluggish and the play store fails to open.
This is on an Acer R11.

**Comment by ksandbergfl** [comment]
This happens on regular/stock Chromebooks too, I have a couple that suffer from this same problem (including an almost-new Acer 315).  The only solution I have come up with is to turn off Google Play, which disables the Linux VM
